onnxscript/function_libs/torch_lib/ops/linalg.py

```diff
@@ -44,10 +44,28 @@ def aten_linalg_cond(self: TensorType, p: Optional[float] = None) -> TensorType:
     raise NotImplementedError()
 
 
-def aten_linalg_cross(self: TensorType, other: TensorType, dim: int = -1) -> TensorType:
-    """linalg_cross(Tensor self, Tensor other, *, int dim=-1) -> Tensor"""
+@torch_op("aten::linalg_cross")
+def aten_linalg_cross(self: TTensor, other: TTensor, dim: int = -1) -> TTensor:
 
-    raise NotImplementedError()
+    zero = op.Constant(value_ints=[0])
+    one = op.Constant(value_ints=[1])
+    two = op.Constant(value_ints=[2])
+    three = op.Constant(value_ints=[3])
+    axes = op.Expand(dim, op.Constant(value_ints=[1]))
```

Comment:
This is converting a scalar to a 1D ... either Unsqueeze or Reshape is better than Expand. (It is more specific, and likely better understand by downstream analyses and optimizations.)